spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/support/MetricStore.java

```diff
@@ -50,17 +57,24 @@
 	 */
 	public MetricStore(MetricsProperties metricsProperties) {
 		this.metricsProperties = metricsProperties;
-		restTemplate = new RestTemplate();
+		ObjectMapper mapper = new ObjectMapper();
```

Comment:
I thought the hateoas-starter from boot already bootstrapped a `_halObjectMapper` bean in the context. Couldn't we reuse that?